src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -33,4 +43,27 @@ public static boolean isNetworkAvailable(final Context context) {
         return connectivityManager.getActiveNetworkInfo() != null && 
             connectivityManager.getActiveNetworkInfo().isConnectedOrConnecting();
     }
+
+    public static void sendFeedEvent(Context context, String category) {
+        Log.d(TAG, "Logging feed event. Category is " + category);
+
+        getTracker(context).send(new HitBuilders.EventBuilder()
+                .setCategory(category)
+                .setAction("click")
+                .build());
+    }
+
+    private static Tracker getTracker(Context context) {
+        if (tracker == null) {
```

Comment:
OK it's not a huge issue @atavism, especially if trackers definitely don't do anything until you tell them to, but just seemed like we could make it a little cleaner.

The other refactoring I can imagine would be to pass an analytics object into the SDK with another method. If the SDK gets that, then it would just use it instead.